packages/calypso-e2e/package.json

```diff
@@ -16,16 +16,16 @@
 	],
 	"license": "GPL-2.0-or-later",
 	"dependencies": {
-		"config": "^1.28.0",
-		"playwright": "1.11.0"
+		"config": "^3.3.6",
+		"playwright": "1.14.0"
```

Comment:
Any reason to not use the `^` qualifier like other dependencies? Same question for `fs-extra`

Reply:
Playwright is pinned out of caution due to a [crash](https://github.com/Automattic/wp-calypso/issues/53955) in our test suite caused by the Chromium browser bundled with Playwright 1.12 and 1.13.

Same story with `fs-extra` - I tested that versions above the current one would cause `EPERM: operation not permitted` issues in TeamCity. This is the last known version that does not trigger this message.

Of course, this isn't a permanent solution, but until pulling from docker private registry works on M1 Mac, I don't have a way to debug the cause of the permission error locally.